recipes-matter/barton-matter-reference/files/BartonProjectConfig.in

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
```

Comment:
Some issue with our scripts here - approving when I see it until I get to examine the copyright tool.